### PR TITLE
Add uproot-browser

### PR DESCRIPTION
**Pull request by chrisburr**
Requires https://github.com/conda-forge/textual-feedstock/pull/7

cc @henryiii

### CODE REVIEW

**Comment by henryiii** [comment]
0.4.0 is about out, and that will fix Python 3.7 & 3.8, which will be nice.

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes, ready for review. Windows is failing due to what appears to be a bug in the package however it's `noarch: python` so it doesn't really matter.

(cc @henryiii in case you want to look into the Windows failure for the next release)

File: recipes/uproot-browser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - uproot-browser = uproot_browser.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - flit-core
```
Comment (BastianZim) [inline recipes/uproot-browser/meta.yaml:24]:
```suggestion
    - flit-core >=3.3
```
https://github.com/scikit-hep/uproot-browser/blob/5c28fd2290fcbb02ba39ac18a7d6677402591084/pyproject.toml#L2

**Review by BastianZim — COMMENTED** [review]
Not super important but might be nice to have.

**Comment by chrisburr** [comment]
Thanks @BastianZim! 🍰

**Comment by henryiii** [comment]
This looks interesting for the windows issue: https://stackoverflow.com/a/52372390/2402816

**Pull request merged**